tsdb/explode.go

```diff
@@ -33,6 +33,12 @@ func EncodeName(org, bucket platform.ID) [16]byte {
 	return nameBytes
 }
 
+// EncodeNameString converts org/bucket pairs to the tsdb internal serialization
+func EncodeNameString(org, bucket platform.ID) string {
+	name := EncodeName(org, bucket)
+	return string(name[:])
```

Comment:
I think this is the appropriate place to do escaping because the ids can contain arbitrary bytes. The storage/engine.go file has a couple TODO's in it because they ad-hoc apply escaping.